### DISCUSSION TITLE
Sapphire Layout

### DISCUSSION THREAD

**Discussion by fluidsworld** [primefaces/community#2038] [PrimeNG Templates] [answered]
Hi,
I just want to ask for the update of the Sapphire Layout.  Is it planned?
Thanks.
Best regards!

**Comment by fluidsworld** [comment, marked as answer]
I read it is legacy and will have no update :( This is very sad.

**Reply by melloware** [reply]
You can update it yourself to PF 13 or 14. It's not that difficult.
